lighthouse-cli/test/smokehouse/lighthouse-runners/devtools.js

```diff
@@ -77,37 +61,19 @@ async function buildDevtools(logs) {
  * @return {Promise<{lhr: LH.Result, artifacts: LH.Artifacts, log: string}>}
  */
 async function runLighthouse(url, configJson, testRunnerOptions = {}) {
-  /** @type {string[]} */
-  const logs = [];
+  if (!buildDevtoolsPromise) buildDevtoolsPromise = buildDevtools();
```

Comment:
Since you've removed the logs here, can you confirm that debugging issues with `build` are shown in someway visibly in the output still? like, is it now hidden before tons of other output whereas before it was more prominent?

example: make `download-devtools.sh` fail with exit code 1. would running `yarn smoke --runner=devtools` alert us to the issue?

Reply:
Yes, for reference this is the output when I did your suggestion 

```
❯❯❯ yarn smoke --runner devtools --debug oopif-scripts
yarn run v1.22.18
$ node lighthouse-cli/test/smokehouse/frontends/smokehouse-bin.js --runner devtools --debug oopif-scripts
Running ONLY smoketests for: oopif-scripts

oopif-scripts smoketest starting…
+ exit 1

oopif-scripts: testing 'http://localhost:10200/oopif-scripts.html'…
nullnullError: Command failed: bash lighthouse-core/test/devtools-tests/download-devtools.sh
    at checkExecSyncError (child_process.js:790:11)
    at execFileSync (child_process.js:827:15)
    at buildDevtools (file:///Users/asraine/lighthouse/lighthouse-cli/test/smokehouse/lighthouse-runners/devtools.js:30:3)
    at runLighthouse (file:///Users/asraine/lighthouse/lighthouse-cli/test/smokehouse/lighthouse-runners/devtools.js:51:53)
    at runSmokeTest (file:///Users/asraine/lighthouse/lighthouse-cli/test/smokehouse/smokehouse.js:180:18)
    at concurrentMapper.runInPool.concurrency.concurrency (file:///Users/asraine/lighthouse/lighthouse-cli/test/smokehouse/smokehouse.js:78:45)
    at ConcurrentMapper.pooledMap (file:///Users/asraine/lighthouse/lighthouse-cli/test/smokehouse/lib/concurrent-mapper.js:99:28)
    at ConcurrentMapper.runInPool (file:///Users/asraine/lighthouse/lighthouse-cli/test/smokehouse/lib/concurrent-mapper.js:120:31)
    at file:///Users/asraine/lighthouse/lighthouse-cli/test/smokehouse/smokehouse.js:78:29
    at Array.map (<anonymous>)
  smoketest results:
  Correctly passed 0 assertions
  Failed 1 assertion
oopif-scripts smoketest complete.

1 assertion failing in total
We have 1 failing smoketest(s): oopif-scripts
internal/process/esm_loader.js:74
    internalBinding('errors').triggerUncaughtException(
                              ^

Error [ERR_SERVER_NOT_RUNNING]: Server is not running.
    at new NodeError (internal/errors.js:322:7)
    at Server.close (net.js:1624:12)
    at Object.onceWrapper (events.js:519:28)
    at Server.emit (events.js:400:28)
    at emitCloseNT (net.js:1677:8)
    at processTicksAndRejections (internal/process/task_queues.js:81:21) {
  code: 'ERR_SERVER_NOT_RUNNING'
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
See https://github.com/GoogleChrome/lighthouse/pull/14205#discussion_r919436905

I'm gonna go with your setup idea